evaluation/metrics/similarity/speaker_similarity.py

```diff
@@ -47,73 +48,131 @@ def extract_speaker_embd(
     return output
 
 
-def extract_speaker_similarity(target_path, reference_path):
-    model = RawNet3(
-        Bottle2neck,
-        model_scale=8,
-        context=True,
-        summed=True,
-        encoder_type="ECA",
-        nOut=256,
-        out_bn=False,
-        sinc_stride=10,
-        log_sinc=True,
-        norm_sinc="mean",
-        grad_mult=1,
-    )
-
-    gpu = False
-    model.load_state_dict(
-        torch.load(
-            "pretrained/rawnet3/model.pt",
-            map_location=lambda storage, loc: storage,
-        )["model"]
-    )
-    model.eval()
-    print("RawNet3 initialised & weights loaded!")
+def extract_similarity(path_ref, path_deg, **kwargs):
```

Comment:
Add the comments for `overall` and `pairwith`.  You can list the typical scenario to exemplify (like [this](https://github.com/open-mmlab/Amphion/tree/main/egs/svc/MultipleContentsSVC#train-from-existing-source)). I'm a little confusing: do we need to choose `overall` when there are NOT ground-truth reference audios, and to choose `pairwith` when there are ground truths?

Reply:
Example added in the egs/metrics/README.md